api/DedicatedWorkerGlobalScope.json

```
@@ -52,40 +52,40 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/close",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "4"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18"
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "12"
             },
             "firefox": {
-              "version_added": "54"
+              "version_added": "3.5"
             },
             "firefox_android": {
-              "version_added": "54"
+              "version_added": "4"
             },
             "ie": {
-              "version_added": null
+              "version_added": "10"
             },
             "opera": {
-              "version_added": true
+              "version_added": "11.5"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "11.5"
             },
             "safari": {
-              "version_added": null
+              "version_added": "4"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "3.2"
```

This fails because the base interface is marked as 5.1, needs to check that one 🤔

---

Hmm. I don't trust that version much, though I'm less confident about mirroring it. It looks like the base interface data was added to BCD during the migration from the wiki. It doesn't look like there was any attempt to verify it at the time. As it stands, it suggests that this API existed on desktop for 2.5 years before making it to iOS. That seems pretty unlikely to me. I guess we could resolve this by setting the parent interface to `"3.2"` for iOS as well.

---

Worker support needs some process/thread work so I wouldn't be surprised if that happened.

[This blog post](https://www.davidbcalhoun.com/2011/new-mobile-safari-stuff-in-ios5-positionfixed-overflowscroll-new-input-type-support-web-workers-ecmascript-5/) says Apple indeed tried enabling it on iOS 3 but then deferred it to 5. Several other posts also report same, so I guess 5.1 sounds right 👀

---

Oh, nice find! Should we go the other direction to resolve the conflict then?

```suggestion
              "version_added": "5.1"
```